VulkanHppGenerator.hpp

```diff
@@ -632,6 +654,14 @@ class VulkanHppGenerator
                                           std::vector<size_t> const & returnParamIndices,
                                           bool                        raii ) const;
   std::string generateConstexprString( std::string const & structName ) const;
+  std::string generateCppModuleConstexprDefines() const;
```

Comment:
Is there a danger that those additional "Usings" function add additional maintenance overhead because they need to be kept in sync?

Maybe the functions generating the symbols could keep track of what needs to be exported and add to an ordered set or vector whenever they generate a symbol that needs to be exported. I'm not sure if that would reduce the amount of added code.

There are also more complex template specializations (` template <typename X, typename Y>
  struct StructExtends`) that probably need to be exported one-by-one (or does `using vk::StructExtends` export all spezializations?). It would be nice if the logic which struct extends which would not need to be duplicated.